### PR TITLE
add delete_live + check_diskspace to bootstrax. Switch to rundb. Misc changes.

### PR DESCRIPTION
**Pull request by jorana**
These are several changes made to bootstrax. Several features were added that were deemed usefull after/during the ceph stress test (https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:dsg:daq:ceph_stress_test).

### CODE REVIEW

**Comment by jorana** [comment]
In https://github.com/XENONnT/straxen/pull/47/commits/f2d101a11fe14d61f23108e25344d946fd8ce84c I've added an option to specify the number of max-messages used for raw_records and records. 

This allows us to limit this number which may turn out to be necessary for high data rates since we have seen that too much RAM is required for such runs. Due to multiprocessing several threads are opened and at some point, this swamps the memory usage. A new process will be opened wherefor the system cannot allocate sufficient memory which causes a crash in strax.

**Comment by jorana** [comment]
As @JelleAalbers pointed out, the implementation in f2d101a of managing the max-messages for raw_records and records cannot be achieved in this manner. Hence. I've reverted those changes in 
2709079.

File: straxen/plugins/daqreader.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ class DAQReader(strax.Plugin):
     dtype = strax.record_dtype()
     parallel = 'process'
     rechunk_on_save = False
+    compressor = 'lz4'
```
Comment (JelleAalbers) [inline straxen/plugins/daqreader.py]: See the discussion in https://github.com/AxFoundation/strax/pull/227: this is necessary for now but hopefully we can find another solution eventually.

File: bin/bootstrax [hunk 2]
```diff
@@ -396,6 +430,24 @@ def log_warning(message, priority='warning'):
...
+    for i in range(wait_diskspace_min_space):
+        if disk_free > wait_diskspace_min_space:
+            # Good we can write, lets continue
+            return
+        else:
+            print(f'Insufficient free disk space ({disk.disk_free} GB)'
+                  f'on {hostname}. Waiting {wait_diskspace_dt} s ({i}th iteration')
+            time.sleep(wait_diskspace_dt)
+            send_heartbeat()
+    assert False, "No diskspace to write to"
```
[the remark this answers is not in the document]
Comment (jorana) [reply bin/bootstrax]: Thanks, I've done so in https://github.com/XENONnT/straxen/pull/59

File: bin/bootstrax [hunk 3]
```diff
@@ -91,23 +106,21 @@ actions.add_argument('--fail', nargs='+',
...
 
 ##
 # Configuration
 ##
 
-# TODO change this to the run database when DAQ commissioning is finished
-# mongo_url = 'mongodb://{username}:{password}@gw:27019/xenonnt'
-# dbname = 'xenonnt'
-# run_collname = 'run'
+print(f'---\nTEST VERSION 0.2.2\n---')
```
[the remark this answers is not in the document]
Comment (jorana) [reply bin/bootstrax]: I'm just keeping track of this for the moment, I want to remove it after we have done all the tests we want to do.

File: bin/bootstrax [hunk 4]
```diff
@@ -76,11 +78,24 @@ parser = argparse.ArgumentParser(
...
                     help="Start strax processes with debug logging.")
+parser.add_argument('--profile', type=str, default='false',
+                    help="Option to run strax in profiling mode. "
+                         "argument specifies the name of the profile if not 'false'. Use e.g. 'date'.prof")
 parser.add_argument('--cores', type=int, default=25,
                     help="Maximum number of workers to use in a strax process. "
                          "Set to -1 for all available cores")
 parser.add_argument('--target', default='event_info',
                     help="Strax data type name that should be produced")
+parser.add_argument('--delete_live', type=str,
```
[the remark this answers is not in the document]
Comment (jorana) [reply bin/bootstrax]: Thanks, I'll look into it

**Pull request merged**